src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Account/Register.cshtml

```diff
@@ -1,36 +1,43 @@
 ﻿@model RegisterViewModel
 @{
-    ViewData["Title"] = "Register";
+    ViewData["Title"] = "Rejestracja";
 }
 
 <h2>@ViewData["Title"]</h2>
-
-<div class="row">
+<div class="row content-centered">
     <div class="col-md-4">
         <form asp-route-returnUrl="@ViewData["ReturnUrl"]" method="post">
-            <h4>Create a new account.</h4>
+            <h4>Stwórz nowe konto.</h4>
```

Comment:
there is no need to display it - remove it

Reply:
Fixed